components/core/src/clp_s/CommandLineArguments.cpp

```diff
@@ -349,6 +364,18 @@ CommandLineArguments::parse_arguments(int argc, char const** argv) {
                 throw std::invalid_argument("ordered-chunk-size must be used with ordered argument"
                 );
             }
+
+            if (m_mongodb_uri.empty() ^ m_mongodb_collection.empty()) {
```

Comment:
Can you add a brief comment on why you use "xor"? like "either be both unspecified, or must be both specified"

Reply:
Sure, I'll add a comment